### DISCUSSION TITLE
/posts/2024/04/28%E5%A6%82%E4%BD%95%E8%AE%A8%E5%A5%BD%E8%AF%BB%E8%80%85%E6%88%96%E8%A7%82%E4%BC%97

### DISCUSSION THREAD

**Discussion by giscus** [Justin3go/justin3go.com#93] [Announcements]
# /posts/2024/04/28%E5%A6%82%E4%BD%95%E8%AE%A8%E5%A5%BD%E8%AF%BB%E8%80%85%E6%88%96%E8%A7%82%E4%BC%97

坚持深耕技术领域的T型前端程序员, 关注独立开发与AI，喜欢Vuejs、Nestjs, 还会点Python、搜索引擎、NLP、Web3、后端

https://justin3go.com/%E5%8D%9A%E5%AE%A2/2024/04/28%E5%A6%82%E4%BD%95%E8%AE%A8%E5%A5%BD%E8%AF%BB%E8%80%85%E6%88%96%E8%A7%82%E4%BC%97

**Comment by dohard-ma** [comment]
![写作公式](https://oss.justin3go.com/blogs/Pasted%20image%2020240427112621.png)

有点像英雄之旅哦
